### ISSUE TITLE
Document FC features

### ISSUE THREAD

**Issue by stanislavlevin** [fleet-commander/fc-admin#211]
There is no document ( manpage, README, etc.) about FC features in details.
Looking at the logger code ```logger/fleet_commander_logger.py``` I see, that we have several high level trackers:
``` python
        GSettingsLogger(self.connmgr)                                           
        NMLogger(self.connmgr)                                                  
        ChromiumLogger(self.connmgr)                                            
        ChromeLogger(self.connmgr)                                              
        FirefoxLogger(self.connmgr)
```
Looks like something is tracking.
But actually, their features fully rely on underlying tracked app.
For example, 
1) I don't know whether FC traces Firefox bookmarks or not unless I check it.
2) Or even more, FC tracked FF bookmarks yesterday, but today not. Assuming that today is my first day with FC I could think - "that's OK, bookmarks are not supported".

So, I mean that we don't have actual and complete list of supported features.
In other words, I don't know what to expect.

**Comment by aruiz** [comment]
At the moment there is only one person working on Fleet Commander full time, we intend to write down docs once we achieve a number of engineering milestones. In the meantime any help would be welcome.
